security/2.0/_index.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: "Jakarta Security 2.0 (milestone)"
-date: 2020-06-11
+title: "Jakarta Security 2.0"
+date: 2020-09-02
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Security defines a standard for creating secure Jakarta EE applications in modern application paradigms.
 
 * [Jakarta Security 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.es/releases/2.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Security 2.0 Specification Document](./security-spec-2.0.0-SNAPSHOT.pdf) (PDF)
-* [Jakarta Security 2.0 Specification Document](./security-spec-2.0.0-SNAPSHOT.html) (HTML)
+* [Jakarta Security 2.0 Specification Document](./jakarta-security-spec-2.0.0.pdf) (PDF)
```

Comment:
There's still a problem, "jakarta" needs to be removed as well.

Reply:
Oh, and sorry, 2.0.0 needs to become 2.0.

Reply:
Ok my bad I misunderstood.
Bean Validation has the jakarta in the name of the file. Same for CDI which I tried to use for consistency.
Happy to yank jakarta- prefix from file name here or to update the Jakarta Security project to add it.

What do you prefer?

I'll update the version at the same time.

Reply:
For the moment, I removed the jakarta prefix here. We can update docs and link later if we want consistency

Reply:
I spot checked a few specs: authentication, authorization, security and expression language, server pages, standard tag library and servlet, but none of these have the jakarta- prefix. So for consistency I think it's best not to have it here either.

Reply:
Then all fine. I removed it